src/components/theme/Login/Login.jsx

```diff
@@ -222,25 +229,7 @@ class Login extends Component {
                       </Grid.Column>
                     </Grid.Row>
                     <Grid.Row stretched>
-                      <Grid.Column stretched width="12">
-                        <p className="help">
-                          <FormattedMessage
-                            id="If you you do not have an account here, head over to the {registrationform}."
-                            defaultMessage="If you you do not have an account here, head over to the {registrationform}."
-                            values={{
-                              registrationform: (
-                                /* eslint-disable jsx-a11y/tabindex-no-positive */
-                                <Link to="/register" tabIndex={1}>
-                                  <FormattedMessage
-                                    id="registration form"
-                                    defaultMessage="registration form"
-                                  />
-                                </Link>
-                              ),
-                            }}
-                          />
-                        </p>
-                      </Grid.Column>
+                      <Grid.Column stretched width="12"></Grid.Column>
```

Comment:
I know this should be conditional on self-registration enabled. Do we have an endpoint for Plone settings?

Reply:
Not yet... it's part of that inglorious endpoint for "bootstrap" some things here and there, we could make it a static config for now.